vuepress/docs/next/tutorials/ecr/deploy-components-without-bundle.md

```diff
@@ -121,67 +118,58 @@ spec:
   entandoPluginNamespace: entando
 ```
 
-Now add this link to your environment
+1. Add this link to your environment
 ```
 kubectl create -f my-link.yaml -n entando
 ```
 
-A new link deployer will start and will work behind the scenes to add your plugin `ingressPath` (this is part of the
-plugin `spec`) to the EntandoApp ingress.
+A new link deployer will start and work behind the scenes to add your plugin `ingressPath` (part of the plugin `spec`) to the EntandoApp ingress.
 
-## 5. Upload the micro frontends to your Entando App
+## Upload the Static Resources 
 
-Now that the plugin and the app are linked together, you can proceed to generate the Micro Frontend from the App Builder
+Now that the plugin and app are linked, generate the Micro Frontend from the App Builder
 and upload the static resources like `js` and `css` files.
 
-1. From AppBuilder go to `Configuration -> File Browser` and create a new folder inside the `public` folder and make the name of the folder the same as the name of the bundle (the value is in the `code` field available in the `/bundle/descriptor.yaml` file of the blueprint project) or using a custom name, e.g. `demo-widget`
+1. From the AppBuilder go to `Configuration -> File Browser` 
+2. Migrate to the `public` folder and create a new folder with the same name as the bundle (the value is in the `code` field available in the `/bundle/descriptor.yaml` file of the blueprint project) or use a custom name such as `demo-widget`.
```

Comment:
Blueprint capitalized
comma for readability - "...project), or use..."

Reply:
There is nothing called a Blueprint project, I think it's just a reference to this project folder. 
I don't think the comma helps because the parenthesis already cuts up the sentence in a heavy handed way?